Assets/Live2D/Cubism/Core/ArrayExtensionMethods.cs

```diff
@@ -252,16 +252,17 @@ internal static unsafe void ReadFrom(this CubismDynamicDrawableData[] self, Cubi
 
 
                 // Copy vertex positions.
+                var vertexLength = data.VertexPositions.Length;
                 fixed (Vector3* dataVertexPositions = data.VertexPositions)
                 {
-                    for (var v = 0; v < data.VertexPositions.Length; ++v)
+                    var vertexPointer = vertexPositions[i].UnmanagedFixedAddress;
+                    for (var v = 0; v < vertexLength; ++v)
                     {
-                        dataVertexPositions[v].x = vertexPositions[i][(v * 2) + 0];
-                        dataVertexPositions[v].y = vertexPositions[i][(v * 2) + 1];
+                        dataVertexPositions[v].x = vertexPointer[(v * 2) + 0];
+                        dataVertexPositions[v].y = vertexPointer[(v * 2) + 1];
```

Comment:
We cannot accept this change, as it would make the existing exception checks in the following section ineffective, which raises safety concerns.
https://github.com/Live2D/CubismUnityComponents/blob/304d0d0c827b7bcb86c6f5d9b0a6398eaccda556/Assets/Live2D/Cubism/Core/Unmanaged/CubismUnmanagedArrayView.cs#L214-L238